### PR TITLE
Add rendring for al niente on hairpins

### PR DESCRIPTION
**Pull request by rettinghaus**
This adds rendering for al niente circles to hairpins.

hairpin-al-niente

(Example from Gould, will be added to test suite.)

### CODE REVIEW

File: src/view_control.cpp [hunk 1]
```diff
@@ -533,40 +533,43 @@ void View::DrawHairpin(
     FloatingPositioner *leftLink = hairpin->GetCorrespFloatingPositioner(hairpin->GetLeftLink());
     FloatingPositioner *rightLink = hairpin->GetCorrespFloatingPositioner(hairpin->GetRightLink());
 
+    const int unit = m_doc->GetDrawingUnit(staff->m_drawingStaffSize);
+    const hairpinLog_FORM form = hairpin->GetForm();
+    const bool niente = (hairpin->GetNiente() == BOOLEAN_true);
```
Comment (lpugin) [inline src/view_control.cpp:538]: It usually better to check if an attribute is not default before getting its value. At least that would be necessary if one day we want to have [this](https://github.com/rism-digital/verovio/issues/2545#issuecomment-997668899) type of optimisation

Comment (lpugin) [reply src/view_control.cpp]: I also meant it for `const hairpinLog_FORM form = hairpin->GetForm();`

Comment (rettinghaus) [reply src/view_control.cpp]: We check for this in line 528.

Comment (lpugin) [reply src/view_control.cpp]: Correct

**Pull request merged**